database.sql

```diff
@@ -845,7 +845,7 @@ CREATE TABLE IF NOT EXISTS `intro` (
 CREATE TABLE IF NOT EXISTS `key-value` (
 	`k` varbinary(50) NOT NULL COMMENT '',
 	`v` mediumtext COMMENT '',
-	`updated_at` int unsigned NOT NULL COMMENT 'timestamp of the last update',
+	`updated_at` int unsigned COMMENT 'timestamp of the last update',
```

Comment:
Shouldn’t the timestamp be initialized on value creation as well?

Reply:
> The licence check is still failing.

And it will fail for this PR at all, because I changed the `daemon.php` :)